docs/building-functions/on_cluster_build.md

```diff
@@ -83,6 +105,21 @@ Please provide credentials for image registry used by Pipeline.
 
 7. To update your Function, commit and push new changes, then run `kn func deploy --remote` again.
 
+## Building a Function on the Cluster with Pipelines as Code
+The second, more useful way to build a Function on your cluster is to use pipelines as code.
+
+Pipelines as code is a feature of Tekton that allows you to define CI/CD for your Function using Tekton `Pipelines` and `PipelineRuns` in files stored in your project on GitHub. These files are used to automatically create a pipeline for a Pull Request or a Push to a branch in the repository. This approach enables automation and repeatability using a Git workflow.
+
+The `func` CLI can generate the Tekton resources for you, adding them to a `.tekton` directory in your project. Commit these files to your project and use them to build your function on the cluster.
+
+1. When you ran `func config git set` earlier, this command also generated a `.tekton/pipeline.yaml` and a `.tekton/pipeline-run.yaml` file in your project directory. If you have not run this command, run it now.
```

Comment:
Hey Lance.. just a sugestion.. what about add a note that the user needs to make sure to include Webhook? and for that he needs handy a valid GitHub token with write permission to Webhooks.
```
kn func config git set
...
? Do you want to configure webhook trigger? Yes
? Please enter the GitHub Personal Access Token:
```